TM1py/Services/CellService.py

```diff
@@ -596,12 +607,38 @@ def execute_mdx_cellcount(self, mdx: str, **kwargs) -> int:
         cellset_id = self.create_cellset(mdx, **kwargs)
         return self.extract_cellset_cellcount(cellset_id, delete_cellset=True, **kwargs)
 
+    def execute_view_dataframe(self, cube_name: str, view_name: str, private: bool = False, top: int = None,
+                               skip: int = None, skip_zeros: bool = True, skip_consolidated_cells: bool = False,
+                               skip_rule_derived_cells: bool = False, **kwargs) -> pd.DataFrame:
+        """ Optimized for performance. Get Pandas DataFrame from an existing Cube View
+        Context dimensions are omitted in the resulting Dataframe !
+        Cells with Zero/null are omitted !
+
+        Takes all arguments from the pandas.read_csv method:
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html
+
+        :param cube_name: String, name of the cube
+        :param view_name: String, name of the view
+        :param private: True (private) or False (public)
+        :param top: Int, number of cells to return (counting from top)
+        :param skip: Int, number of cells to skip (counting from top)
+        :param skip_zeros: skip zeros in cellset (irrespective of zero suppression in MDX / view)
+        :param skip_consolidated_cells: skip consolidated cells in cellset
+        :param skip_rule_derived_cells: skip rule derived cells in cellset
+        :return: Pandas Dataframe
+        """
+        cellset_id = self.create_cellset_from_view(cube_name=cube_name, view_name=view_name, private=private, top=top,
+                                                   skip=skip, skip_zeros=skip_zeros,
+                                                   skip_consolidated_cells=skip_consolidated_cells,
+                                                   skip_rule_derived_cells=skip_rule_derived_cells, **kwargs)
+        return self.extract_cellset_dataframe(cellset_id, **kwargs)
```

Comment:
Please pass the `skip` arguments to the `extract_cellset_dataframe` instead of `create_cellset_from_view`.